plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -407,7 +407,7 @@ public boolean isNull(ResultSet resultSet, int columnIndex)
             public long readLong(ResultSet resultSet, int columnIndex)
                     throws SQLException
             {
-                return resultSet.getObject(columnIndex, LocalDate.class).toEpochDay();
+                return requireNonNull(resultSet.getObject(columnIndex, LocalDate.class)).toEpochDay();
```

Comment:
If we’re expecting an NPE with a specific message, let’s change the test, or modify this code to throw a better exception (maybe SQLException with a known error message).